tracer/test/snapshots/Iast.CommandInjection.AspNetCore2.IastEnabled.verified.txt

```diff
@@ -0,0 +1,87 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/executecommand,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Tags: {
+      aspnet_core.route: iast/executecommand,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/executecommand,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/ExecuteCommand?file=nonexisting.exe&argumentLine=arg1,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "COMMAND_INJECTION",
+      
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "nonexisting.exe",
+            "source": 0
+          },
+          {
+            "value": " "
+          },
+          {
+            "value": "arg1",
+            "source": 1
+          }
+        ]
+      }
+    }
+  ],
+  "sources": [
+    {
+      "origin": "http.request.parameter.value",
+      "name": "file",
+      "value": "nonexisting.exe"
+    },
+    {
+      "origin": "http.request.parameter.value",
+      "name": "argumentLine",
+      "value": "arg1"
+    }
+  ]
+},
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
```

Comment:
Should we raise the sampling priority when you detect a vulnerability

Reply:
We have not done that, but it could be an interesting topic to discuss. I will ask other teams.